src/bulk/common.ts

```diff
@@ -294,7 +294,7 @@ export class BulkWriteResult {
   }
 
   toString(): string {
-    return `BulkWriteResult(${this.result})`;
+    return `BulkWriteResult(${EJSON.stringify(this.result)})`;
```

Comment:
I'm happy to revert this  but the new eslint rules actually pointed out that this uses the default stringification (`[object Object]`).